tests/scripts/DataLoaderTests.js

```diff
@@ -0,0 +1,191 @@
+/*!
+ * GPII Preferences Data Loader Tests
+ *
+ * Copyright 2016-2017 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = fluid || require("infusion"),
```

Comment:
These only run on the server so the proper include style is simply var fluid = require("infusion") - similarly for the rest of these files